### DISCUSSION TITLE
Uncaught Too deep hierarchy of inlined custom previews

### DISCUSSION THREAD

**Discussion by Alecyrus** [vuejs/pinia#1135] [Help and Questions]
### Reproduction

```javascript
import { defineStore } from 'pinia';

export const useWindowStateStore = defineStore('windowState', {
  state: () => ({
    isFocused: true,
  }),
  actions: {
    commit() {
      console.log(this);
    },
  },
});
```

```javascript
const windowState = useWindowStateStore();
windowState.commit();
```
result: 
<img width="951" alt="截屏2022-03-10 12 17 38" src="https://user-images.githubusercontent.com/13865581/157588558-c4016f1f-1d40-4dcb-8fc2-9921b45e2d4d.png">

Is it expected? So many nested Reactive obejct.

**Comment by MiguelRen** [comment]
Tenía la esperanza de que aquí estuviese la respuesta... pero nada... que raro que no la hayan respondido si la colocaron hace un año.... Si alguien sabe que pasa en esa situacion.. por favor y gracias.
